root.tf

```diff
@@ -309,3 +311,33 @@ module "api_update_lambda" {
   api_url                               = module.consignment_api.api_url
   keycloak_backend_checks_client_secret = data.aws_ssm_parameter.keycloak_backend_checks_client_secret.value
 }
+
+module "file_format_lambda" {
+  source                                = "./tdr-terraform-modules/lambda"
+  project                               = var.project
+  common_tags                           = local.common_tags
+  lambda_file_format                    = true
+  auth_url                              = module.keycloak.auth_url
+  api_url                               = module.consignment_api.api_url
+  keycloak_backend_checks_client_secret = data.aws_ssm_parameter.keycloak_backend_checks_client_secret.value
+  file_system                           = module.file_format_efs.file_system
+  file_format_efs_access_point          = module.file_format_efs.access_point
+  vpc_id                                = module.shared_vpc.vpc_id
+}
+
+module "file_format_efs" {
+  source            = "./tdr-terraform-modules/efs"
+  common_tags       = local.common_tags
+  function          = "file-format-efs"
+  project           = "tdr"
```

Comment:
Should this be var.project instead of the hardcoded tdr value?

Reply:
Yep, you're right, I've changed it.